content/blog/charts/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Magic Charts
```

Comment:
Any ideas for the title?

Excalicharts?
Magic Charts?
Import charts in seconds?